index.js

```diff
@@ -378,11 +378,27 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = null
+    var versionMsg = null
+
+    if (arguments.length === 1) {
+      version = opt
+      versionOpt = 'version'
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else if (arguments.length === 2) {
+      versionOpt = opt
+      version = msg
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else {
```

Comment:
Personally I like the first option better, I'm going to tend to just specify the option and the version #, and rarely change the description. If I ever want to change the description I'm fine with just specifying three values.

Rather than accepting an object, which I think starts to over complicate things for such a simple option., I like @nexdrew's idea that we'd default to the version # in the nearest package.json (@sindresorhus has a module for finding this file in a consistent way). 

Once we have this 99% of the time I feel people will write:

```js
.version()
// or
.version('v')
// or
.version('v', '99.9')
```

I don't think this would avoid needing to set an object as a parameter or a null description.